src/components/Favicon/Auto/hooks/useCorsProxy.js

```diff
@@ -0,0 +1,5 @@
+export const fixCorsError = (url) => {
+	return `https://corsproxy.io/?${url}`;
+};
+
+export default () => ({ fixCorsError });
```

Comment:
Why do we need `default` import here? You have already exported `fixCorsError` function above this line.

Reply:
You are right, it is better to stick with one exporting method.
Moved everything favicon-grabbing-related into a `FaviconGrabberService`